drivers/filesystems/fastfat/string.c

```diff
@@ -24,3 +25,10 @@ vfatIsLongIllegal(
 {
     return wcschr(long_illegals, c) ? TRUE : FALSE;
 }
+
+BOOLEAN
+IsDotOrDotDot(UNICODE_STRING File)
```

Comment:
Oops, forgot this the first time around, sorry. Passing `UNICODE_STRING`s by value is a bit strange since it's a structure larger than pointer size. I'd normally expect this function to take a `PCUNICODE_STRING` as its argument.

Reply:
Fixed. Thanks.